src/sage/tests/startup.py

```diff
@@ -8,7 +8,7 @@
     sage: from sage.tests.cmdline import test_executable
     sage: environment = "sage.all"
     sage: cmd = f"from {environment} import *\nprint('IPython' in sys.modules)\n"
-    sage: print(test_executable(["sage", "--python"], cmd)[0])  # long time
+    sage: print(test_executable(["python3"], cmd)[0])  # long time
```

Comment:
Maybe?

```suggestion
    sage: print(test_executable(["python"], cmd)[0])  # long time
```

At least `python` is used in the other PR to invoke python.

Reply:
Sure, I just used `python3` because that's what `sage -python` does but I guess there's no ambiguity nowadays